bindings/python/repair_wheel.py

```diff
@@ -0,0 +1,80 @@
+import glob
+import os
+import sys
+from subprocess import run
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+wheel_dir, wheel_file, delocate_args = sys.argv[1:]
+wheel_dir = wheel_dir.replace(os.sep, "/")
+wheel_file = wheel_file.replace(os.sep, "/")
+
+# Ensure pyarrow.
+if "universal2" in wheel_file:
+    macos_ver = os.environ.get("MACOSX_DEPLOYMENT_TARGET", "10.3")
+    macos_ver = macos_ver.replace(".", "_")
+    target = os.path.expanduser("~/wheels")
```

Comment:
Using the home directory (`~`) seems off.

Reply:
I'll use a tempdir here